src/components/DatePicker/DatePicker.jsx

```diff
@@ -35,6 +35,7 @@ export const DatePicker = ({
     components = {},
     getTooltip,
     upcomingDates,
+    timezone = null, // seller timezone (e.g. "America/Los_Angeles") to return correct today date
```

Comment:
Call it `timezoneName` so it's explicit that it's a name, not an offset or any other variation